assets/scss/thema/lustrum.scss

```diff
@@ -1,81 +1,211 @@
+$body-bg: $kleur-secundair;
+$body-color: $kleur-text;
+$headings-color: $kleur-textheader;
+
+$zijbalk-bg: $kleur-primair;
+$zijbalk-border: $kleur-text-hover;
+$zijbalk-kopje: $kleur-textheader;
+
+$zijbalk-ishetal-ja: $lustrum-kleur-wit;
+$zijbalk-ishetal-nee: $lustrum-kleur-licht-rood;
+
+$bb-block-bg: $kleur-primair;
+
+$card-bg: $kleur-primair;
+$list-group-bg: $kleur-primair;
+
+$link-color: $kleur-text;
+$link-hover-color: $kleur-text-hover;
+
+$menu-primary-hover: $kleur-textheader;
+$menu-bg: $body-bg;
+$menu-color: $body-color;
+$menu-small-bg: $body-bg;
+$menu-small-color: $body-color;
 
-$headings-font-family: 'Orpheus Pro', 'Source Sans Pro', Helvetica, Arial, sans-serif;
+$zoeken-active-bg: lighten($body-bg, 10%);
 
-@import '../base';
+$tabel-header: $kleur-primair;
+$tabel-hover: darken($kleur-secundair, 5%);
+$tabel-regel-licht: $kleur-secundair;
+$tabel-regel-donker: lighten($kleur-secundair, 5%);
 
-body {
-	background-color: $lustrum-kleur-rood;
+$footer-background: $kleur-primair;
+$maaltijdketzer-bg: $kleur-primair;
+
+// overige variabelen in dies theme
+$bb-alt-bg: darken($body-bg, 4%);
+$bb-hover-bg: $kleur-text-hover;
+
+$modal-content-bg: $kleur-secundair;
+$list-group-action-color: $kleur-text;
+$list-group-hover-bg: $kleur-textheader;
+$text-muted: #fffbfa;
+
+/* importeer basis */
+@import "../base";
+
+// Puntjes op de i (maak het onleesbare leesbaar)
+
+// headers
+h1 {
+	font-family: $headings-font-family;
+	color: $kleur-textheader;
 }
 
-#menu {
-	background: $lustrum-kleur-donker-rood;
-	border-bottom-color: $lustrum-kleur-goud;
-	h2 {
-		color: $lustrum-kleur-goud;
-	}
-	#cd-primary-nav {
-		.nav-link {
-			color: $lustrum-kleur-goud;
-		}
-	}
+h2, h3, h4, h5, h6 {
+	text-transform: uppercase;
+	font-family: $headings-font-family;
+	color: $kleur-textheader;
 }
 
+// zijbalk links
 #zijbalk .blok .zijbalk-kopje a {
 		font-family: $headings-font-family;
-		color: $lustrum-kleur-licht-goud;
 		text-transform: uppercase;
+		color: $kleur-textheader;
 }
 
-#zijbalk {
-	background-color: $lustrum-kleur-donker-rood;
-	.blok .item {
-		color: $lustrum-kleur-goud;
-		a {
-			color: $lustrum-kleur-goud;
-		}
-	}
+// lustrum stek link
+.lustrum12_button {
+	color: $kleur-text;
+}
+.lustrum12_button:hover {
+	color: $kleur-text-hover;
+}
+
+// overwrite header colors
+#menu #cd-primary-nav .nav-link{
+	color: $lustrum-kleur-goud;
+}
+
+// image in footer, make white
+footer img{
+	filter: invert(90%);
 }
 
-.bb-block {
-	box-shadow: $lustrum-kleur-goud;
+// name selection right top
+#menu-personal{
+	color: $kleur-text;
 }
 
-.ishetal:after {
-	content: '';
-	display: block;
-	height: 156px;
-	background: url('../../images/lustrum12/Lustrum12.png') no-repeat center center;
-	background-size: contain;
-	margin: 20px 0;
-	clear: left;
+#instellingen .list-group-item:hover {
+	background-color: $kleur-primair;
+	color: $kleur-text;
 }
 
-.ishetal {
-	.nee {
-		color: $lustrum-kleur-rood;
+// knopjes
+.btn-secondary {
```

Comment:
Wil je niet gewoon `$secondary` zetten? Dan heb je alle plekken te pakken waar secondary wordt gebruikt.

Reply:
Ik heb er nog even naar gekeken, maar als ik een secondary definieer, veranderen er gelijk heel veel andere dingen mee. Dus ik denk dat het voor nu wel goed is.